src/test/java/edu/hw1/Task5Test.java

```diff
@@ -0,0 +1,34 @@
+package edu.hw1;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import static org.assertj.core.api.AssertionsForClassTypes.assertThat;
+
+public class Task5Test {
+    Task5 task = new Task5();
+    @Test
+    @DisplayName("Palindrome number should return true")
+    void checkPalindromeNumber(){
```

Comment:
`BUG` в джаве методы тестов(и только тестов) принято делать в  snake_case
```suggestion
    void check_palindrome_number(){
```

Reply:
Что насчет названия по типу Should_ReturnTrue_When_GivenPalindromeNumber ?

Reply:
норм, только с маленькой буквы

Reply:
а еще если есть такое осмысленное название метода, то можно не писать аннотации DisplayName?

Reply:
Да, но я больше люблю осмысленное описание в **@DisplayName** и не очень название метода, но тут лишь вкусовщина